lib/testers/rule-tester.js

```diff
@@ -69,8 +69,8 @@ const RuleTesterParameters = [
     "code",
     "filename",
     "options",
-    "args",
-    "errors"
+    "errors",
```

Comment:
Should we also validate the properties of error objects in tests? For example, someone could accidentally use `lineEnd` instead of `endLine` and errors wouldn't be caught.